SpatialGDK/Documentation/content/glossary.md

```diff
@@ -486,11 +486,11 @@ Once you’ve chosen a label for the worker type (for example, myWorkerType), yo
 
 Zoning is one of the multiserver options for working with SpatialOS (the other option is [offloading](#offloading)). It involves splitting up the world into zones, known as “areas of [authority](#authority)”, with a different [server-worker instance](#server-workers) responsible for each. A server-worker instance can make updates only to Actors that are in its area of authority.
```

Comment:
```suggestion
Zoning is one of the multiserver options for working with SpatialOS (the other option is [offloading](#offloading)). It involves splitting up the world into areas of [authority](#authority), with a different [server-worker instance](#server-workers) responsible for each. A server-worker instance can make updates only to Actors that are in its area of authority.
```

This is a shameless driveby change of something that I missed when I was making updates yesterday! 😳